tests/unit_tests/test_lib/test_service_port_file.py

```diff
@@ -30,6 +30,20 @@ def finished_process():
     return proc
 
 
+def test_reads_unix_token(tmp_path, running_process):
```

Comment:
maybe add a test that has both unix and tcp in the port file?

Reply:
that's this test

Reply:
don't you only write the unix part

and i see that in the go code theoretically we could write both unix and tcp

Reply:
ugh sorry... i have a brain fog, i just looked at the first of sock=
and didn't parse to the end of the string

Reply:
haha emberssing 🤦